app/code/Magento/Ui/view/base/web/js/form/form.js

```diff
@@ -277,7 +277,9 @@ define([
             var invalidField = _.find(this.delegate('checkInvalid'));
 
             if (!_.isUndefined(invalidField) && _.isFunction(invalidField.focused)) {
-                invalidField.focused(true);
+                setTimeout(function() {
+                    invalidField.focused(true);
```

Comment:
@VladimirZaets 
https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Ui/view/base/web/js/form/components/fieldset.js#L167

fieldset.js open that tab then focus on element will work

Hope it will make sense

can you please confirm my js changes loaded?

Reply:
@VladimirZaets  please check it now

Reply:
@sunilit42 So maybe we should fix it globally, because, looks like have the same ( wrong ) behavior in all cases where we have dropdown and validation

Reply:
@VladimirZaets  then i can add if more than one tab then use mode code otherwise as default one

Reply:
@sunilit42, it's not a valid case in general, we should do an action when some action happened, we should investigate how we can manage this situation. In your case if someone will change the configuration of dropdown ( time of opening to 2 sec for example) this bug again will be reproducible. 

Otherwise, if we need to move call this function to the end of the call stack we need to make setTimeout 0.

Reply:
There are a lot of places Magento itself using timeout

Reply:
@sunilit42 a lot of legacy code in Magento that currently, we don't recommend to use.